README.md

```diff
@@ -10,19 +10,60 @@ An Express REST API that mocks a minimal implementation of a centralized exchang
 
 ## Getting Started
 
+First, clone this repository locally and move into it:
 ```
-# start a postgres db
-docker run \
-  -p 5432:5432 \
-  --name portal-exchange-db \
-  -e POSTGRES_PASSWORD=password \
-  postgres
 
-# install dependencies
-yarn
+git clone https://github.com/portal-hq/portalex.git && cd portalex
 
-# start the server
-CUSTODIAN_API_KEY=<YOUR_CUSTODIAN_API_KEY> \
-  WEBHOOK_URL=<YOUR_WEBHOOK_URL> \
-  yarn demo
 ```
+
+Then check if you have anything running on ports 3000, 5432, and 4040:
```

Comment:
love this note, but it's safe to assume that a developer will be responsible for making sure they're not running anything other servers. So we can remove lines 20 - 29.